Library/Homebrew/cli/args.rb

```diff
@@ -5,12 +5,48 @@
 module Homebrew
   module CLI
     class Args < OpenStruct
+      attr_accessor :processed_options
```

Comment:
Does this need to be public?

Reply:
`processed_options` are set from inside `CLI::Parser` class

https://github.com/Homebrew/brew/pull/6479/files#diff-e0e03670c451d934ae1d79adfff63cacR142

Reply:
Gotcha!